addons/web_editor/static/lib/odoo-editor/test/spec/list.test.js

```diff
@@ -3018,20 +3018,20 @@ describe('List', () => {
                             await testEditor(BasicEditor, {
                                 contentBefore: '<ol><li>abc</li><li>[]def</li></ol>',
                                 stepFunction: deleteBackward,
-                                contentAfter: '<ol><li>abc[]def</li></ol>',
+                                contentAfter: '<ol><li>abc</li></ol><p>[]def</p>',
                             });
                             // With another list item after.
                             await testEditor(BasicEditor, {
                                 contentBefore: '<ol><li>abc</li><li>[]def</li><li>ghi</li></ol>',
                                 stepFunction: deleteBackward,
-                                contentAfter: '<ol><li>abc[]def</li><li>ghi</li></ol>',
+                                contentAfter: '<ol><li>abc</li></ol><p>[]def</p><ol><li>ghi</li></ol>',
```

Comment:
@Zinston Same here. It basically splits the list in two. However, Google Docs does it in a smart way that does keep the numbering. Here, the numbering will reset at 1 for "ghi", which is probably not what we want.

Reply:
Same comment as above, this seems inconsistent to me. Should we get our PO involved here actually?